_osl/osl.env

```diff
@@ -0,0 +1,33 @@
+# Export these vars in your terminal session before running pnpm commands.
+
+# set -a
+# source osl.env
+# set +a
+
+# Use digests to update the manifests accordingly
+USE_IMAGE_DIGESTS=true
+# Won't build any images in this repo, just the manifests. The images are built within internal systems.
+KIE_TOOLS_BUILD__buildContainerImages=false
+
+# SHA required for the operator image. If you don't have it yet, just comment it.
+OSL_OPERATOR_IMAGE__buildTag=sha256:ffae74d76431b428341abfe8b8b704323261c224ff9140a2ea2782c0524f222f
+
+SONATAFLOW_OPERATOR__registry=registry.redhat.io
+SONATAFLOW_OPERATOR__account=openshift-serverless-1
+SONATAFLOW_OPERATOR__name=openshift-serverless-1-logic-rhel8-operator
+SONATAFLOW_OPERATOR__buildTag=sha256:ffae74d76431b428341abfe8b8b704323261c224ff9140a2ea2782c0524f222f
+
+SONATAFLOW_OPERATOR__kogitoDataIndexPostgresqlImage=registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel8:1.36.0
+SONATAFLOW_OPERATOR__kogitoDataIndexEphemeralImage=registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel8:1.36.0
+SONATAFLOW_OPERATOR__kogitoJobsServicePostgresqlImage=registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.36.0
+SONATAFLOW_OPERATOR__kogitoJobsServiceEphemeralImage=registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8:1.36.0
+SONATAFLOW_OPERATOR__sonataflowDevModeImage=registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:1.36.0
+SONATAFLOW_OPERATOR__sonataflowBuilderImage=registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:1.36.0
+SONATAFLOW_OPERATOR__kogitoDBMigratorToolImage=registry.redhat.io/openshift-serverless-1/logic-db-migrator-tool-rhel8:1.36.0
+
+# If Drools version is different, set the Drools version manually in the `maven-base/pom.xml`
+KOGITO_RUNTIME_version=9.103.0.redhat-00003
+SONATAFLOW_DEVMODE_IMAGE__sonataflowQuarkusDevUiVersion=9.103.0.redhat-00003
+SONATAFLOW_QUARKUS_DEVUI_VERSION=9.103.0.redhat-00003
```

Comment:
again is this always the same as `KOGITO_RUNTIME_version` value?

Reply:
Not always. Upstream differs, so that's why we have different versions here.